ready_set_go.md

```diff
@@ -90,4 +90,4 @@ $ mlatu script "100 101 max"
 101
 ```
 
-Lots of people who come from imperative languages tend to stick to the notion that parentheses should denote function application. For example, in C, you use parentheses to call functions like `foo()`, `bar(1)`, or `baz(3, "haha")`. Like we said, spaces are used for function applicaation in Mlatu. So those functions in Mlatu would be `foo`, `1 bar`, and `"haha" 3 baz`.
\ No newline at end of file
+Lots of people who come from imperative languages tend to stick to the notion that parentheses should denote function application. For example, in C, you use parentheses to call functions like `foo()`, `bar(1)`, or `baz(3, "haha")`. Like we said, spaces are used for function applicaation in Mlatu. So those functions in Mlatu would be `foo`, `1 bar`, and `"haha" 3 baz`.
```

Comment:
Some minor nitpicks: 
"tend to stick to the notion" -> "tend to have the notion"
"Like we said" -> "As mentioned earlier"
"aplicaation in Mlatu" -> "application in Mlatu"

Reply:
Fixed and pushed!